js/views/account.js

```diff
@@ -65,6 +65,10 @@ define(function(require) {
 		toggleCollapse: function() {
 			this.collapsed = !this.collapsed;
 			this.render();
+		},
+		toggleMenu: function(e) {
+			e.preventDefault();
+			this.menuShown = !this.menuShown;
 			this.listenTo(Radio.ui, 'document:click', function(event) {
```

Comment:
Hm, doesn't that mean everytime the menu is toggled, another event handler is registered?